source/braille.py

```diff
@@ -2042,6 +2044,9 @@ def __init__(self):
 		self._cursorPos = None
 		self._cursorBlinkUp = True
 		self._cells = []
+		# Queue of objects to use in L{handleUpdate}. Length 1 keeps
+		# recent object and discards others.

```

Comment:
What is the point of having a queue? Why not simply updating an attribute to a new object?